index.d.ts

```diff
@@ -89,5 +93,6 @@ declare module 'react-native-document-picker' {
     ): Promise<DocumentPickerResponse[]>;
     static isCancel<IError extends { code?: string }>(err?: IError): boolean;
     static releaseSecureAccess(uris: Array<string>): void;
+    static pickDirectory(): Promise<DirectoryPickerResponse> | null;
```

Comment:
maybe it should just be Promise<string> ? what do you prefer?

Reply:
hello, please keep this as-is. An object is easier to extend if needed in future.